src/Quantity.elm

```diff
@@ -516,22 +516,24 @@ sqrt (Quantity value) =
 
 
 {-| Cube a quantity with some `units`, resulting in a new quantity in
-`Cubed units.
+`Cubed units`.
```

Comment:
:+1: elm-format messed with this comment and I must've put it back wrong. Thanks.

Reply:
I think it's OK as it is. `Squared units` referred to in the same way. Technically maybe `Squared` `units` when used in a description rather than code - but reads OK like this.

Reply:
Yup, I think `Cubed units` is best - the function takes `Quantity number units` and returns `Quantity number (Cubed units)`.